src/Symfony/Component/EmojiTransliterator/README.md

```diff
@@ -0,0 +1,14 @@
+EmojiTransliterator Component
+=============================
+
+The EmojiTransliterator component provides an ext-intl Transliterator that can
+transliterate emojis to ASCII.
+
+Resources
+---------
+
+ * [Documentation](https://symfony.com/doc/current/components/emoji_transliterator.html)
```

Comment:
This page needs to be created in `symfony/symfony-docs` afterwards

Reply:
I created an issue https://github.com/symfony/symfony-docs/issues/18171

Reply:
I created a PR to create that page